lib/home_screen.dart

```diff
@@ -14,7 +16,11 @@ class _HomeScreenState extends State<HomeScreen> {
   int selectedIndex = 0;
   static const TextStyle optionStyle =
```

Comment:
seems like an unused field please remove it

Reply:
all changes done... @dk-a-dev